### DISCUSSION TITLE
Add mobile phone support

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5815] [Feature Requests]
Other collection management software support entering data using a mobile phone. This is largely dependent on supporting Safari (https://github.com/specify/specify7/projects/24), but there are a number of things that should be improved.

This enhancement could include
- fields should be full-width on mobile
- dialogs need to be full-screen as well with intuitive controls
- barcode scanning needs to supported to insert text into fields
- query interface needs to be simple for phones

Some compromises (such as not supporting WorkBench on mobile) would be expected.

**Comment by maxpatiiuk** [comment]
@grantfitzsimmons can I merge this with #3387 or #2826?

**Comment by grantfitzsimmons** [comment]
@maxpatiiuk That's fine with me!
